apps/web/src/pages/Pools/IndexPoolPage/components/PoolTable.tsx

```diff
@@ -67,6 +67,21 @@ function dateAccessor(value: Maybe<string>, intl: IntlShape) {
   ) : null;
 }
 
+const fullNameCell = (pool: Pool, intl: IntlShape) => {
+  return (
+    <span>
+      {getFullNameHtml(pool.owner?.firstName, pool.owner?.lastName, intl)}
+    </span>
+  );
+};
+
+const classificationsCell = (pills: JSX.Element[] | null) => {
+  if (pills) {
+    return <span>{pills}</span>;
+  }
```

Comment:
`<Pill />` should be wrapped in `<Pills />` to show the proper styling when there is more than one.

Reply:
we don't appear to have such a component, it'll need to be made

![image](https://user-images.githubusercontent.com/40485260/217303460-ad072435-f856-4ccb-b4c1-1e7e74e5044e.png)

and added to other places like here

![image](https://user-images.githubusercontent.com/40485260/217302493-e866d997-6d62-4d90-9bdc-f6913f5d9291.png)

Reply:
Oh, you are right. Shoot! Umm, okay. I'd say out of scope. I can create an issue for that 👍

Reply:
thanks!